backends/magma/ceed-magma.h

```diff
@@ -21,11 +21,34 @@
 
 #define USE_MAGMA_BATCH
 #define USE_MAGMA_BATCH2
+#define USE_MAGMA_BATCH3
 
 void magma_dtensor_contract(Ceed ceed,
                             CeedInt A, CeedInt B, CeedInt C, CeedInt J,
                             const CeedScalar *t, CeedTransposeMode tmode,
                             const CeedInt Add,
                             const CeedScalar *u, CeedScalar *v);
 
+int t30_setup(void *ctx, CeedInt Q, const CeedScalar *const *in,
+              CeedScalar *const *out);
+int t30_mass( void *ctx, CeedInt Q, const CeedScalar *const *in,
+              CeedScalar *const *out);
+int t20_setup(void *ctx, CeedInt Q, const CeedScalar *const *in,
+              CeedScalar *const *out);
+int t20_mass(void *ctx,  CeedInt Q, const CeedScalar *const *in,
+              CeedScalar *const *out);
+int ex1_setup(void *ctx, CeedInt Q, const CeedScalar *const *in,
+              CeedScalar *const *out);
+int ex1_mass(void *ctx,  CeedInt Q, const CeedScalar *const *in,
+             CeedScalar *const *out);
+int t400_setup(void *ctx, CeedInt Q, const CeedScalar *const *in,
+              CeedScalar *const *out);
+int t400_mass(void *ctx,  CeedInt Q, const CeedScalar *const *in,
+             CeedScalar *const *out);
+int t500_setup(void *ctx, CeedInt Q, const CeedScalar *const *in,
+               CeedScalar *const *out);
+int t500_mass(void *ctx,  CeedInt Q, const CeedScalar *const *in,
+              CeedScalar *const *out);
+
 #define CeedDebug(...)
+//#define CeedDebug(format, ...) fprintf(stderr, format, ## __VA_ARGS__)
```

Comment:
Does this need to be disabled like this?

Reply:
No. I don't remember any reason to do it.

Reply:
Why was the issue marked "resolved" without that debugging feature being restored?